gravitee-apim-console-webui/src/services-ngx/debug-api.service.ts

```diff
@@ -37,7 +37,11 @@ export class DebugApiService {
     },
   ): Observable<Event> {
     // clean endpoint http proxy
+    // eslint:disable-next-line:@typescript-eslint/ban-ts-comment
+    // @ts-ignore
     if (api.proxy && api.proxy.endpoints) {
+      // eslint:disable-next-line:@typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
:o

Reply:
🤷‍♂️  I prefer not to touch this code. and to review together later

Reply:
but if you tell me to "delete everything there is no risk". i put all this to 🚮